02-introducing-survey-data.Rmd

```diff
@@ -1,2 +1,127 @@
+# Overview of Surveys {#c02}
+
+```{r}
+#| include: false
+
+library(markdown)
+library(webshot)
+library(magick)
+if (webshot::is_phantomjs_installed() == FALSE) {
+  webshot::install_phantomjs() 
+}
+```
+
+Creating and fielding surveys to provide estimates of a population are much more complex than simply adding a few questions to an online platform. Survey researchers can spend months or even years developing the study design, questions, and other methods for a single survey to ensure high quality data is collected. While this book focuses on the analysis methods of surveys, understanding the entire survey process can provide a better understanding of what types of analyses should be conducted on the data. There are many pieces of existing literature that cover designing and implementing surveys that we recommend reviewing in detail (e.g., {cite: Dillman book, Groves book, education book?, others?}). The information provided in this chapter is a brief introduction into the survey process aiming to provide a foundation of the complexities of survey data.
+
+The survey life cycle starts with a research topic or question of interest. For example, a researcher may be interested in the impact childhood trauma has on political affiliation later in life. Before choosing to conduct a survey, researchers typically review existing data sources to determine if data are already available that can answer this question. If existing data cannot answer the nuances of the research question, a survey can be used to capture the exact data that the researcher needs.
+
+Figure \@ref(fig:svylifecycle) shows a high level view of the steps that researchers must conduct to go from a research question to final results.
+
+<!-- Note: need to adjust the size of the text in this figure. -->
+
+```{r svylifecycle, echo=FALSE, fig.cap = "Overview of the survey process"}
+library(DiagrammeR)
+mermaid("
+graph TD
+  A[Survey Concept]-->B[Study Design]
+  B-->C[Questionnaire Design]
+  B-->D[Data Collection Materials]
+  C-->E[Collect Data]
+  D-->E
+  E-->F[Create Weights]
+  F-->G[Analysis]
+  G-->H[Reporting]
+  
+  style A fill: #bfd7ea, stroke: #0b3954
+  style B fill: #bfd7ea, stroke: #0b3954
+  style C fill: #bfd7ea, stroke: #0b3954
+  style D fill: #bfd7ea, stroke: #0b3954
+  style E fill: #bfd7ea, stroke: #0b3954
+  style F fill: #bfd7ea, stroke: #0b3954
+  style G fill: #bfd7ea, stroke: #0b3954
+  style H fill: #bfd7ea, stroke: #0b3954
+")
+```
+
+For some studies, this may be a cycle where after the first wave of the study, the next one begins shortly after and the process begins again.  Other studies may require translation as part of questionnaire design and data collection materials preparation, or include pretesting such as cognitive and usability testing prior to data collection.  Each study is slightly different how it is developed, but this process provides a general flow for how surveys are developed.
+
+## Survey Preparation
```

Comment:
Perhaps make sections that briefly discuss each box and follow the same flow so a second level heading for each box in diagram. They will have varying lengths but still touch on weighting and resources, for example. For analysis/reporting, that's most of this book